Content.Client/UserInterface/Systems/Atmos/GasTank/GasTankWindow.cs

```diff
@@ -27,12 +29,13 @@ public sealed class GasTankWindow
         private readonly RichTextLabel _lblInternals;
         private readonly Button _btnInternals;
 
-        public GasTankWindow(GasTankBoundUserInterface owner)
+        public GasTankWindow(GasTankBoundUserInterface owner, IEntityManager entityManager)
         {
             TextureButton btnClose;
             _resourceCache = IoCManager.Resolve<IResourceCache>();
             _owner = owner;
-            var rootContainer = new LayoutContainer {Name = "GasTankRoot"};
+            _entityManager = entityManager;
+            var rootContainer = new LayoutContainer { Name = "GasTankRoot" };
```

Comment:
Just use IoCManager.Instance and resolve both manually as the entmanager isn't used outside of here.